Services/Mail/classes/Address/Parser/RFC822.php

```diff
@@ -891,10 +891,6 @@ protected function _validateLocalPart($local_part)
 
         // Validate each word.
         foreach ($words as $word) {
-            // word cannot be empty (#17317)
```

Comment:
@iszmais Instead of removing this block, could you please use comments and surround this block with something like:
```php
// mjansen patch 16 Sep 2015 start

// ...Code

// mjansen patch 16 Sep 2015 end
```